graphql/resolvers/mutation.go

```diff
@@ -125,13 +166,19 @@ func (r mutationResolver) OpenBug(_ context.Context, input models.OpenBugInput)
 	}, nil
 }
 
-func (r mutationResolver) CloseBug(_ context.Context, input models.CloseBugInput) (*models.CloseBugPayload, error) {
-	b, err := r.getBug(input.RepoRef, input.Prefix)
+func (r mutationResolver) CloseBug(ctx context.Context, input models.CloseBugInput) (*models.CloseBugPayload, error) {
+	repo, b, err := r.getBug(input.RepoRef, input.Prefix)
 	if err != nil {
 		return nil, err
 	}
 
-	op, err := b.Close()
+	id := identity.ForContext(ctx, repo)
+	if id == nil {
+		return nil, ErrNotAuthenticated
+	}
+	author := cache.NewIdentityCache(repo, id)
```

Comment:
This bypass the repo level caching and will break the uniqueness of an identity in memory. See my other comment on having the identity ID in the context.